Taskfile.yml

```diff
@@ -158,3 +158,20 @@ tasks:
     desc: Accept all test snapshots. (Shortcut to accept snapshots after running tests)
     cmds:
       - Rscript -e 'testthat::snapshot_accept()'
+
+  install-air:
+    desc: Install Air to format R code.
+    cmds:
+      - curl -LsSf https://github.com/posit-dev/air/releases/latest/download/air-installer.sh | sh
```

Comment:
This is downloaded every time `task fmt` runs, which is bad. @eitsupi do you know why this is executed while other setup tasks are skipped? Is it because this is a curl command and there's no way to know if it's outdated?